torch/_dynamo/output_graph.py

```diff
@@ -25,13 +25,14 @@
 
 import torch.nn
 from torch import fx
+from torch._guards import Guard
 from torch.fx.experimental.symbolic_shapes import ShapeEnv
 
 from . import config, logging as torchdynamo_logging, variables
 from .bytecode_transformation import create_instruction, Instruction, unique_id
 from .codegen import PyCodegen
 from .exc import BackendCompilerFailed, unimplemented
-from .guards import Guard, GuardBuilder
+from .guards import GuardBuilder
```

Comment:
Consider reexporting Guard from guards so you don't have to wobble all these sites?

Reply:
I dislike that pattern, I prefer to just point to where things are, not chaining them as transitive deps.